.php_cs

```diff
@@ -5,7 +5,6 @@ $finder = Symfony\CS\Finder\DefaultFinder::create()
 ;
 
 return Symfony\CS\Config\Config::create()
-    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
```

Comment:
Because this function doesn't exist in Php-cs-fixer 0.* which is needed to test sf 2.3.